packages/api-client/src/libs/send-request.ts

```diff
@@ -264,18 +267,21 @@ export const createRequestOperation = <ResponseDataType = unknown>({
     })
 
     // Populate all forms of auth to the request segments
-    Object.keys(example.auth).forEach((k) => {
-      const exampleAuth = example.auth[k]
-      const scheme = securitySchemes[k]
+    request.selectedSecuritySchemeUids.forEach((uid) => {
+      const exampleAuth = auth[uid]
+      const scheme = securitySchemes[uid]
       if (!exampleAuth || !scheme) return
 
       // Scheme type and example value type should always match
       if (scheme.type === 'apiKey' && exampleAuth.type === 'apiKey') {
         const value = replaceTemplateVariables(exampleAuth.value, env)
-        if (scheme.in === 'header') headers[scheme.nameKey] = value
-        if (scheme.in === 'query') urlParams.append(scheme.nameKey, value)
-        if (scheme.in === 'cookie')
-          Cookies.set(scheme.nameKey, value, cookieParams)
+        if (scheme.in === 'header') headers[exampleAuth.name] = value
+        if (scheme.in === 'query') urlParams.append(exampleAuth.name, value)
+        if (scheme.in === 'cookie') {
+          Cookies.set(exampleAuth.name, value)
```

Comment:
Yea not sure what the plan is with cookies, but at least if you remove the params you can see them in dev tools (otherwise they don't show).

We should log em on the void server to test if they are working

Reply:
Void Server should respond with cookies already: https://github.com/scalar/scalar/blob/main/packages/void-server/src/utils/getRequestData.ts#L44

It’s just that they are blocked for security reasons in most scenarios. 😢